src/program/snabbnfv/neutron2snabb/neutron2snabb.lua

```diff
@@ -96,8 +96,13 @@ function filter (port, secbindings, secrules, direction)
          end
       end
    end
-   if #rules > 0 then return { rules = rules }
-                 else return nil end
+   if type == "stateless" then
```

Comment:
Ok I am pretty sure this branch will break this as of now. E.g. if we look at the code referenced above, even in case of `secbindings[port.id]` being false an empty filter config will be returned. I assume that `--no-security-groups` leads exactly to this case, so the config should only be returned when `secbindings[port.id]` is a true value, e.g. the return conditional should be moved up into `if secbindings[port.id] then`?